src/test/java/org/folio/edge/orders/MainVerticleTest.java

```diff
@@ -458,4 +480,39 @@ public void testPlaceOrderTimeout(TestContext context) throws MalformedApiKeyExc
         new ErrorWrapper("REQUEST_TIMEOUT", MSG_REQUEST_TIMEOUT));
     assertEquals(respBody.toXml(), resp.body().asString());
   }
+
+  @Test
+  public void testShouldReturnEmptyResponseIfResponseFromVendorIsEmpty() {
+    String body = StringUtils.EMPTY;
+
+    final Response resp = RestAssured
+      .with()
+      .body(body)
+      .post("/orders?type=GOBI&apiKey=" + apiKey)
+      .then()
+      .statusCode(201)
+      .extract()
+      .response();
+
+    assertEquals(StringUtils.EMPTY, resp.body().asString());
+  }
+
+  @Test
+  public void testShouldReturnErrorWithInternalFormatIfAcceptAndContentHeaderAreDifferent() {
+    logger.info("=== Test place order - Success (JSON) ===");
+
+    String PO = "118279";
+    String body = mockRequests.get(PO);
+
+    final Response resp = RestAssured
+      .with()
+      .accept(APPLICATION_JSON)
+      .body(BODY_REQUEST_FOR_HEADER_INCONSISTENCY)
+      .post("/orders?type=GOBI&apiKey=" + apiKey)
+      .then()
+      .contentType(APPLICATION_JSON)
+      .statusCode(400)
+      .extract()
+      .response();
+  }
```

Comment:
Maybe add a test that covers the cases I mentioned earlier
* `*/*`
* multiple values
* w/ encoding